mutable_tree.go

```diff
@@ -156,8 +156,8 @@ func (tree *MutableTree) Get(key []byte) ([]byte, error) {
 //
 // Import can only be called on an empty tree. It is the callers responsibility that no other
 // modifications are made to the tree while importing.
-func (tree *MutableTree) Import(version int64) (*Importer, error) {
-	return newImporter(tree, version)
+func (tree *MutableTree) Import(version int64, traverseOrder OrderType) (*Importer, error) {
```

Comment:
Same as https://github.com/cosmos/iavl/pull/662#discussion_r1092063909, I'm not sure we should break the Import API here.  `ImportPreOrder` might be better.